### PR TITLE
Homework 6

### PR DESCRIPTION
**Pull request by xo4yxypmy**
Check my homework pls

### CODE REVIEW

File: homework6/homework6.cpp [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+    int f1 = 1, f2 = 1, next;
+
+    for (int i = 3; i <= nF; ++i) {
+        next = f1 + f2;
+        f1 = f2;
+        f2 = next;
+    }
+
+    std::cout << "Element number " << nF << " in the Fibonacci sequence: " << (nF > 2 ? f2 : 1) << std::endl;
```
Comment (bohdanpc) [inline homework6/homework6.cpp]: f2 is initialized with 1 anyway, so this check is redundant.

File: homework6/homework6.cpp [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+            std::cout << "*";
+        }
+        std::cout << std::endl;
+        vidstup++;
+    }
+    std::cout << "e" << std::endl;
+
+    for (int a = 0; a < f; a++) {
+        for (int b = 0; b <= a; b++) {
+            std::cout << (b % 2);
```
Comment (bohdanpc) [inline homework6/homework6.cpp]: the correct pattern is different. 
your output will always start with the same digit on each row

**Review by bohdanpc — CHANGES_REQUESTED** [review]

File: homework6/homework6.cpp [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+    int f1 = 1, f2 = 1, next;
+
+    for (int i = 3; i <= nF; ++i) {
+        next = f1 + f2;
+        f1 = f2;
+        f2 = next;
+    }
+
+    std::cout << "Element number " << nF << " in the Fibonacci sequence: " << f2 << std::endl;
```
Comment (xo4yxypmy) [inline homework6/homework6.cpp:36]: changed

File: homework6/homework6.cpp [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+            std::cout << "*";
+        }
+        std::cout << std::endl;
+        vidstup++;
+    }
+    std::cout << "e" << std::endl;
+
+    for (int a = 0; a < f; a++) {        
+        for (int b = 0; b <= a; b++) {
+            std::cout << (a + b + 1) % 2;
```
Comment (xo4yxypmy) [inline homework6/homework6.cpp:107]: changed: now the loop has the same pattern as in the task

**Review by helen-black — APPROVED** [review]

**Pull request merged**